packages/core/src/loginUtils.ts

```diff
@@ -6,6 +6,9 @@ import { parse, stringify } from 'query-string';
 import { HttpCall, HttpQueryParams } from './httpClient/basicHttpClient';
 import { LogoutUrlResponse } from './types';
 import isString from 'lodash/isString';
+import { ClientCredentials, ClientOptions } from './baseCogniteClient';
+// eslint-disable-next-line lodash/import-scope
```

Comment:
Please don't ignore linting rules.
This rule was added explicitly here: https://github.com/cognitedata/cognite-sdk-js/blob/master/.eslintrc.js#L30
We did a cleanup of this some time ago: https://github.com/cognitedata/cognite-sdk-js/pull/386